cmd/influx_stress/example.toml

```diff
@@ -0,0 +1,82 @@
+[write]
+  concurrency = 10
+  batch_size = 5000
+  batch_interval = "0s"
+  database = "stress"
+  precision = "n"
+  address = "localhost:8086"
+  reset_database = true
+  # doesnt do anything
+  starting_point = "6m" # how far back in time to go
+
+[[series]]
+  point_count = 10 # number of points that will be written for each of the series
+  measurement = "cpu"
+  series_count = 100000
+
+  tag_count = 20 # 
```

Comment:
currently nothing, it should have a `# doesnt do anything` comment.

Eventually the idea is that you'll be able to specify a series with a large number of tags. For example, `tag_count=3` will generate series of the form

```
cpu,tag-key-0=tag-value-0,tag-key-1=tag-value-0,tag-key-2=tag-value-0 value=100
```